packages/terra-graphs/tests/wdio/Graphs/BarGraph/bar-graph-spec.js

```diff
@@ -2,64 +2,64 @@ Terra.describeViewports('BarGraph', ['tiny', 'medium', 'large'], () => {
   describe('Basic bar graphs', () => {
     it('validates Axis info text labels graph container', () => {
       browser.url('/raw/tests/cerner-terra-graphs-docs/graphs/bar-graph/basic-bar-graphs/axis-info-text-labels');
-      Terra.validates.screenshot('axis_info_text_labels', { selector: '.carbon-graph-container' });
+      Terra.validates.screenshot('axis_info_text_labels');
```

Comment:
Does selectors has any issue with node 16 update ..??

Reply:
As this task is also WIP and blocked It was just for testing. The same reason is for your question regarding deleting WDIO snapshots. Here is described: https://jira2.cerner.com/browse/UXPLATFORM-6962